site/index.html

```diff
@@ -119,33 +143,89 @@ <h2>Sample NGS features</h2>
                         </li>
                     </ul>
                 </li>
+                <li>
+                    and much, much more :)
+                </li>
             </ul>
 
-        </div>
-        <div class="col-sm">
-            <h2>Documentation</h2>
+            <h2 class="text-left">Links</h2>
             <ul>
-                <li><a href="doc/latest/index.html">Documentation Index</a> - main entry point to all documentation</li>
+                <li><a href="http://tut.ngs-lang.org" target="_blank">Examples-based tutorial</a> - understand how NGS feels like</li>
```

Comment:
We can't have `http` links on this site in 2021. It's either `https` or not to have them at all (or find another solution entirely).

Reply:
Looking into this. will take a while, because AWS xD

Reply:
@ilyash Https is now available to the endpoints.   Please look at https://github.com/ngs-lang/ngs/pull/486/commits/df620518be7890f1f641bc76c485a75f7a2ddd46 since i had to change the way the main.css is referenced so that https://ref.ngs-lang.org can refer to it.
An alternative could be to copy the main.css to the `generated` folder and then have a different function for the headers for the `generated` folder. Not sure if there is another way, can you provide help?